install/config.EXAMPLE.php

```diff
@@ -96,7 +96,7 @@
 const STATS_LOCATION = ROOT_LOCATION . 'stats/';
 const ASSETS_LOCATION = ROOT_LOCATION . 'assets/';
 const CACHE_LOCATION = ASSETS_LOCATION . 'cache/';
-const LIBS_LOCATION = ASSETS_LOCATION . 'libs/';
+const LIBS_LOCATION = ASSETS_LOCATION . 'libs/@bower_components/';
```

Comment:
Can't yarn simply include the dependencies inside `libs`?  Not that big of a deal but it would be nicer this way.

Reply:
Maybe yes, I did not find way how to do it. That `@bower_components` directory is some magic directory from `bower-away` migration...